From: Dan Carpenter <dan.carpenter@oracle.com>
Date: Fri, 11 Jan 2013 09:57:33 +0300
Subject: [PATCH 003/108] tg3: missing break statement in
 tg3_get_5720_nvram_info()
Patch-mainline: v3.9-rc1
Git-commit: 17e1a42f2e377ec5809af79eb651d9ed5958fb97
References: bsc#908458 FATE#317507


There is a missing break statement so FLASH_5762_EEPROM_HD gets treated
like FLASH_5762_EEPROM_LD.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 drivers/net/ethernet/broadcom/tg3.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/ethernet/broadcom/tg3.c
+++ b/drivers/net/ethernet/broadcom/tg3.c
@@ -13802,8 +13802,10 @@ static void tg3_get_5720_nvram_info(stru
 		switch (nvmpinstrp) {
 		case FLASH_5762_EEPROM_HD:
 			nvmpinstrp = FLASH_5720_EEPROM_HD;
+			break;
 		case FLASH_5762_EEPROM_LD:
 			nvmpinstrp = FLASH_5720_EEPROM_LD;
+			break;
 		}
 	}
 
